ykman/cli/info.py

```diff
@@ -27,10 +27,29 @@
 
 from __future__ import absolute_import
 
-from ..util import APPLICATION
+from ..descriptor import open_device
+from ..fido import FipsU2fController
+from ..oath import OathController
+from ..otp import OtpController
+from ..util import APPLICATION, TRANSPORT
 import click
 
 
+def get_overall_fips_status(serial):
+    stati = {}
```

Comment:
`stati` ?

Reply:
Meh, seems like [my latin fu failed me](https://english.stackexchange.com/questions/877/what-is-the-plural-form-of-status).